test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG base_tag=1.0.6.0-linux-arm32v7
-FROM edgebuilds.azurecr.io/microsoft/azureiotedge-module-base-full:${base_tag}
+FROM azureiotedge/azureiotedge-module-base-full:${base_tag}
```

Comment:
please make sure all base images exist in docker hub, especially module base full.

Reply:
Yeah, I'm going through building the images right now. 
It should fail then if an image doesn't exist.